src/precice/SolverInterface.cpp

```diff
@@ -290,9 +290,9 @@ void SolverInterface:: writeBlockScalarData
 
 void SolverInterface:: writeScalarData
 (
-  int    dataID,
-  int    valueIndex,
-  double value )
+  int           dataID,
+  int           valueIndex,
+  const double& value )
```

Comment:
These changes should not be part of this PR. Similar to https://github.com/precice/precice/pull/494/files#r320836465

Reply:
I created a PR for adding these changes independently: https://github.com/precice/precice/pull/530

Reply:
0615ee2 removes these changes from this PR.